### DISCUSSION TITLE
Provide Elsa 2.3.X over NuGet

### DISCUSSION THREAD

**Discussion by ItsDyze** [elsa-workflows/elsa-core#2556] [Q&A] [answered]
Hi! I'm working on a .net 3.1 version and struggle to get a version including the fix from #1126 on nuget.
Would it be possible to get a sweet 2.3.0.1 with the fix on it?

**Comment by sfmskywalker** [comment, marked as answer]
2.4 is right around the corner, but going forward I will be creating release branches so we can push out fixes more swiftly.

**Reply by ItsDyze** [reply]
Thanks! I built it locally for now.
